src/popup/popupInterfacesSOAP.js

```
@@ -0,0 +1,61 @@
+        // Send getSOAPInterfaces message to the content script
+        chrome.tabs.sendMessage(
+          tab.id,
+          { greeting: "getSOAPInterfaces" },
+          function (response) {
+            if (chrome.runtime.lastError) {
+              console.error("Error sending message:", chrome.runtime.lastError);
+            } else if (response) {
+              console.log("Received response:", response);
+              // Process the response as needed
+              // save the code response to a file pulling thefilename frome the last part of the URL and the code from the response
+              const filename = response.url.split("/").pop() + ".xml";
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)